playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-upload-flow.spec.ts

```diff
@@ -51,6 +57,15 @@ test.describe.serial('Blood Kits upload flow', () => {
   for (const study of studies) {
     test(`Should upload a single kit for one participant @functional @dsm @${study}`, async ({page}, testInfo) => {
       testResultDir = testInfo.outputDir;
+      if (clinicalStudies.includes(study)) {
```

Comment:
Good spot for ternary operator

Reply:
`isClinicalStudy = clinicalStudies.includes(study) ? true : false;`

Reply:
I tried that before but it seems to make lint complain:
<img width="669" alt="Screenshot 2024-06-03 at 4 24 04 PM" src="https://github.com/broadinstitute/ddp-angular/assets/5404082/60682226-b8ab-44aa-802c-fc8e3ab16e47">